scripts/packetReader.lua

```diff
@@ -336,6 +336,45 @@ packetReader.GetActorPacketTables = function(packetType)
                 actor.killer.uniqueIndex = tes3mp.GetActorKillerRefNum(packetIndex) ..
                     "-" .. tes3mp.GetActorKillerMpNum(packetIndex)
             end
+        elseif packetType == "ActorAI" then
+            actor.ai = {}
+
+            local action = tes3mp.GetActorAIAction(packetIndex)
+            actor.ai.action = action
+
+            if action == enumerations.ai.ACTIVATE or action == enumerations.ai.COMBAT or action == enumerations.ai.ESCORT then
```

Comment:
Missing enumerations.ai.FOLLOW, those actors currently only show their index and the requested action.